pkg/cmd/experimental/router/router.go

```diff
@@ -93,22 +93,22 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 			case 1:
 				name = args[0]
 			default:
-				glog.Fatalf("You may pass zero or one arguments to provide a name for the router")
+				fmt.Errorf("You may pass zero or one arguments to provide a name for the router")
```

Comment:
Actually since this is a command, it should follow the pattern we use everywhere else, ie. factor out `Run` as a separate function named `Run<NameOfTheCommand>` and simply return errors when we encounter one. Back, the caller should use `cmdutil.CheckErr`.